rental.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+#  create class Rental
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(_data)
```

Comment:
- [ ] Let's correct the syntax error _on line 8_ by updating the constructor parameter to `date` instead of `_data`.